src/components/Navbar/NavbarElements.js

```diff
@@ -94,10 +99,49 @@ export const NavBtnLink = styled(LinkR)`
   cursor: pointer;
   transition: all 0.2s ease-in-out;
   text-decoration: none;
-
   &:hover {
     transition: all 0.2s ease-in-out;
     background: #fff;
     color: #010606;
   }
 `;
+
+export const PlBtn = styled.button`
```

Comment:
name styled components more properly (don't know what is P1Btn or UkBtn), this need to be informative